### DISCUSSION TITLE
Roadmap

### DISCUSSION THREAD

**Discussion by darrekk** [jellyfin/Swiftfin#1127] [Questions] [answered]
I'm just wondering if there are any plans published anywhere? I would like to contribute but I am just not sure what is worth working on. This is the first open source project I've worked on so I'm not really sure what the process is supposed to look like. There isn't a ticket system or anything, right? @LePips is there anything specific you would like to see done? Or should I just comment on bugs to see if they are worth fixing?

**Comment by LePips** [comment, marked as answer]
Thank you for your interest in contributing to the project! You're correct, I don't have a ticketing system because _that's a job_ and I don't care right now to maintain anything like that. Per the contributing guidelines, just comment on things that you would like to work on or create issues and draft PRs for features that you would like to pitch for feedback. I will typically comment when someone says that they would like to do something if there is prerequisite work needed or if the feature isn't ready to be implemented yet.
